packages/terra-time-input/src/_Minute.jsx

```diff
@@ -0,0 +1,67 @@
+class Minute {
+  static NUMBERS = [...Array(60).keys()];
+
+  static STRINGS = [...Minute.NUMBERS.map((num) => String(num).padStart(2, 0)), ...Minute.NUMBERS.map((num) => String(num))];
+
+  /**
+   * Factory method to create a Minute from a string. Use this to take form input and normalize it for later operations.
+   * @param {String} string A string expressing an minute. Can be be like '1' or '01'.
+   * @returns a Minute object or undefined if the minute doesn't make sense.
+   */
+  static FromString(string) {
+    if (!Minute.STRINGS.includes(string)) {
+      // throw new Error(`Expected a string in ${Minute.STRINGS}`);
```

Comment:
is there a reason this was commented out?

Reply:
I will delete it. It's left over from an earlier approach where I thought raising an error was a good idea. I changed to returning `undefined` instead.